pkg/option/option.go

```diff
@@ -109,30 +110,51 @@ type Options struct {
 
 // addClusterVars introduces cluster arguments.
 func addClusterVars(opt *Options) {
-	opt.flags.StringVar(&opt.ClusterName, "cluster-name", "eg-cluster-default-name", "Human-readable name for the new cluster, ignored while joining an existed cluster.")
-	opt.flags.StringVar(&opt.ClusterRole, "cluster-role", "primary", "Cluster role for this member (primary, secondary).")
-	opt.flags.StringVar(&opt.ClusterRequestTimeout, "cluster-request-timeout", "10s", "Timeout to handle request in the cluster.")
+	opt.flags.StringVar(&opt.ClusterName, "cluster-name", "eg-cluster-default-name",
```

Comment:
per https://github.com/golang/go/wiki/CodeReviewComments#line-length:

> don't add line breaks to keep lines short when they are more readable long--for example, if they are repetitive.

please revert the newly added line breaks.